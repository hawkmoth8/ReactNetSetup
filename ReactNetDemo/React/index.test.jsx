import CommentBox from "./index";

describe("index tests", () => {
  it("should render", () => {
    const wrapper = shallow(<CommentBox />);
    expect(wrapper).toMatchSnapshot();
  });
});
