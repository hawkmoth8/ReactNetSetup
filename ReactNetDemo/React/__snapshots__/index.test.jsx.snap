// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index tests should render 1`] = `
<Fragment>
  <div
    className="blue-text"
  >
    Hello, world! I am a CommentBox.....!!
  </div>
  <img
    alt=""
    src="/images/dog-paw.svg"
  />
  <img
    alt=""
    src="/images/400x300.png"
  />
</Fragment>
`;
